throughput/spam.go

```diff
@@ -435,3 +436,10 @@ func (s *Spammer) returnFunds(ctx context.Context, cli *jsonrpc.JSONRPCClient, p
 	)
 	return nil
 }
+
+func sleep(it *time.Timer, start time.Time) {
+	// Determine how long to sleep
+	dur := time.Since(start)
+	sleep := max(float64(consts.MillisecondsPerSecond-dur.Milliseconds()), 0)
+	it.Reset(time.Duration(sleep) * time.Millisecond)
+}
```

Comment:
```suggestion
// sleep updates the timer to tick immediately if >= 1s has elapsed
// and otherwise tick after 1s has elapsed since start
func sleep(it *time.Timer, start time.Time) {
	dur := time.Since(start)
	sleep := max(float64(consts.MillisecondsPerSecond-dur.Milliseconds()), 0)
	it.Reset(time.Duration(sleep) * time.Millisecond)
}
```